src/main/clj/parti_time/input/api.clj

```diff
@@ -2,15 +2,12 @@
 
 
 (defn file-extension [filename]
-  (re-find #"(?<=\.)[^.]+$" filename))
+  (or (re-find #"(?<=\.)[^.]+$" filename)
+      "tl"))
 
 (defmulti read-timeline
   file-extension)
 
-(defmethod read-timeline nil [filename]
```

Comment:
The default dispatch is not needed any more, since our dispatch function fills in the default value "tl" for nil.